client/cmdlfhid.c

```diff
@@ -431,7 +431,7 @@ static int CmdHIDBrute(const char *Cmd) {
                 cmdp++;
                 break;
             default:
-                PrintAndLogEx(WARNING, "Unknown parameter '%c'", param_getchar(Cmd, cmdp));
+                PrintAndLogEx(WARNING, _YELLOW_("Unknown parameter '%c'"), param_getchar(Cmd, cmdp));
```

Comment:
how about just coloring the param part '%c' , not the text.   Makes it easy to spot the important part.